tests/util/factory.js

```diff
@@ -1,3 +1,9 @@
+import {
+  NavigationItem,
+  NavigationTree,
+  NavigationTreeItem
+} from "@reactioncommerce/plugin-navigation/src/simpleSchemas.js";
```

Comment:
@aldeed this works, just wondering if this is how we want to import simpleSchemas from outside packages.

Reply:
@kieckhafer This would be fine. I had kind of hoped that when we got to this we'd start moving the integration tests out of this repo and into their respective package repos, but we have some work to do yet before that can happen.